example.yaml

```diff
@@ -1,9 +0,0 @@
-otk.version: "1"
-
-otk.define.tag:
-  var: 1
-
-otk.target.osbuild.example:
-  pipelines:
-    - otk.include.tag: include.yaml
-    - otk.include.bar: include.yaml
```

Comment:
Can we have the removals of these examples as a separate commit?  They're not related to the arg parser are they?

Reply:
Correct, they shouldn't have been in the project at all.